assets/src/application/ui/layout/entityList/filterBar/buttons/CardViewFilterButton.tsx

```diff
@@ -13,16 +13,16 @@ const CardViewFilterButton: React.FC<CardViewFilterButtonProps> = ({ listId, set
 	const filterId = `ee-card-view-btn-${listId}`;
 
 	return (
-		<EspressoButton
-			buttonType={EspressoButtonType.MINIMAL}
+		<EspressoIconButton
 			className={className}
-			icon={<AppstoreFilled />}
+			icon={() => <AppstoreFilled />}
```

Comment:
maybe a separate PR to copy the svg paths from `@ant-design/icons`

Reply:
Yes